mvn/Makefile

```diff
@@ -0,0 +1,37 @@
+OWNER    = boozallen
+REPO     = sdp-images
+IMAGE    = mvn
+VERSION  = 3.8.5
```

Comment:
I feel like we should be tagging this with both the JDK and Maven version since both are important.